clang/test/CodeGen/ubsan-trap-merge.c

```diff
@@ -0,0 +1,106 @@
+// NOTE: Assertions have mostly been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+// The most important assertion is the attributes at the end of the file, which
+// shows that ubsan attaches 'nomerge' to each ubsantrap intrinsic.
+//
+// RUN: %clang -fsanitize=signed-integer-overflow -S -emit-llvm -fsanitize-trap=all -O3 -mllvm -ubsan-unique-traps %s -o - \
```

Comment:
should use clang_cc1 and an explicit triple

Reply:
Done! I'd overlooked the clang driver distinction - thanks for your electrifying comment!